docs/articles/expensify-classic/settings/Set-Notifications.md

```diff
@@ -4,72 +4,66 @@ description: This article is about how to troubleshoot notifications from Expens
-     - A misspelled email address during account creation.
-     - Use of a distribution list email (acting as an "alias" email) without a linked inbox.
-     - An auto-responder that has been responding to our emails for an extended period.
-- To resolve this issue, confirm that the email address is indeed associated with an active inbox. Then, click the link that says "here," and your email should be unblocked shortly.
-- SMTP Error (Gray Box): In some cases, you might encounter a gray box with an SMTP error message. This error can vary, but it typically looks something like this:
+## How can I be sure that emails from Expensify are legitimate and not spam?
 
-![ExpensifyHelp_SMTPError]({{site.url}}/assets/images/ExpensifyHelp_SMTPError.png){:width="100%"}
+Expensify's emails are SPF and DKIM-signed, meaning they are cryptographically signed as coming from us and are not spam.
 
-**These look a bit cryptic, yes, but hang in there!** 
+## Why do legitimate emails from Expensify sometimes end up marked as spam?
 
-The error messages you see are the raw message text received from your email provider's server to Amazon. These messages can vary in text, but the best course of action is to follow the link provided (by copying and pasting) in the text for the next steps.
+This issue can arise because Expensify send mails from a cloud-based service. The sender’s IP serves multiple vendors, including Expensify. If another vendors using the same IP is marked as spam, it can block all messages from that IP, even if they’re from different vendors (like Expensify).
 
-**Scenario 1**: If the message in the gray box includes "mimecast.com": It means that our emails are being blocked by the server. In this case, you should contact your IT person or team to address the issue.
+## What is the best way to ensure emails are not accidentally marked as Spam? 
 
-**Scenario 2**: If the message in the gray box mentions "blacklist at org/.com/.net," or resembles the screenshot provided, it indicates that your IT team has configured your email to use a third-party email reputation or blacklisting service. Here's what you need to know:
-- All our emails are SPF and DKIM-signed, meaning they are cryptographically signed as coming from us and are not spam.
-- The problem arises because we send mail from a cloud-based service. This means that the sender's IP serves multiple vendors, including Expensify. If one of those vendors is marked as spam, it can block all messages from that IP, even if they're from different vendors (including us).
-- The better approach is for the server to flag spam via DKIM and SPF (rather than solely relying on the sender's IP address), as our messages are correctly signed and encrypted to prevent spoofing.
+For server-level spam detection, it’s best to use DKIM and SPF verification rather than solely relying on the sender’s IP address. Expensify's emails are signed and encrypted to prevent spoofing.
```

Comment:
```suggestion
For server-level spam detection, the safest approach to allowlisting email from Expensify is to verify DKIM and SPF, rather than solely relying on the third-party reputation of the sending IP address.
```